docs/_Resources/01-glossary.md

```diff
@@ -49,6 +50,7 @@ redirect_from: /Resources
 * **Narrative description:** detailed, written description of computational analyses.
 
 ## O
+* **Ontology:** a list of terms with curated textual definitions or synonyms with the additional aspect of relational and hierarchical correlation to other terms.
```

Comment:
THanks for this contribution @magelm , I would split the definition in two to make it more digestible, with a part on the terms (and adding the missing additional aspect is the persistent identifiers for each term ;) ) and a part on the the relationships between the terms.

Even if many relationships in ontologies are hierarchical, not all are. I would be against using the term correlation here that could be confusin